crates/astria-sequencer-validation/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "astria-sequencer-validation"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+ct-merkle = "0.1"
```

Comment:
This is imported from crates.io, but a git-dep of `ct-merkle` is a workspace dep.

Reply:
yeah the one in the workspace is a patch

Reply:
Cool and sorry, didn't notice.